### DISCUSSION TITLE
AttributeError: module 'neurokit2' has no attribute 'ppg_peaks'

### DISCUSSION THREAD

**Discussion by PhD-GOAT** [neuropsychology/NeuroKit#860] [Q&A]
https://neuropsychology.github.io/NeuroKit/functions/hrv.html

When trying to analyze HRV with ppg data, the function states that there is no ppg_peaks. peaks = nk.I wonder if I can do the same as ecg data regardless of info when processing with ppg_findpeaks(data["PPG"], sampling_rate=100).

**Comment by DominiqueMakowski** [comment]
I'm not sure I understand your question, could you rephrase?

**Reply by PhD-GOAT** [reply]
Oh sorry. I've just checked that it's Peaks to return. So even if the way to get peaks from ppg and ecg is different, do I just need to do the same hrv analysis code in ppg like the ecg example?

**Reply by DominiqueMakowski** [reply]
yup
